### PR TITLE
Release 1.6.2

### PR DESCRIPTION
**Pull request by davidgrayston**
### Fixed
- Document details parsing: now accommodates more values _(including redacted Aadhar numbers)_

### CODE REVIEW

File: lib/yoti/data_type/document_details.rb [hunk 1]
```diff
@@ -51,28 +51,20 @@ class DocumentDetails
...
-    end
-
     #
     # Parses provided value into separate attributes
     #
     # @param [String] value
     #
     def parse_value(value)
-      attributes = value.split(' ')
+      attributes = value.split(/\s/)
```
Comment (emmas-yoti) [inline lib/yoti/data_type/document_details.rb]: Do we really want to split on any whitespace character? I'd assume this'd be a breaking change from Connect if it started passing us back tab-separated values?

File: sonar-project.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
 sonar.projectKey = yoti-web-sdk:ruby
 sonar.projectName = ruby-sdk
-sonar.projectVersion = 1.6.1
+sonar.projectVersion = 1.6.2
```
Comment (emmas-yoti) [inline sonar-project.properties]: Not relevant to this PR but I think going forward we plan to change development branch's version-strings to the next/proposed version in the PR that updates development after a release.

File: lib/yoti/data_type/document_details.rb (hunk 1 above)
Comment (emmas-yoti) [reply lib/yoti/data_type/document_details.rb]: > Do we really want to split on any whitespace character? I'd assume this'd be a breaking change from Connect if it started passing us back tab-separated values?
> 
> Alternatively do we want to change the other SDKs to match any ws?

File: sonar-project.properties (hunk 2 above)
Comment (davidgrayston) [reply sonar-project.properties]: Fixed in f26aefe

File: lib/yoti/data_type/document_details.rb (hunk 1 above)
Comment (davidgrayston) [reply lib/yoti/data_type/document_details.rb]: Fixed in f26aefe

**Pull request merged**